packages/orbit-components/src/Card/CardSection/index.tsx

```diff
@@ -105,42 +121,45 @@ export default function CardSection({
           {actions && <Actions actions={actions} />}
         </div>
       )}
-
       {(title != null || header != null) && !expandable && (
-        <div className="p-400 lm:p-600 w-full">
-          <Header
-            title={title}
-            titleAs={titleAs}
-            description={description}
-            expandable={expandable}
-            header={header}
-            expanded={opened}
-            actions={actions}
-            isSection
-          />
+        <div className="gap-300 p-400 lm:p-600 flex items-center ">
+          <ContentWrapper onClick={!children ? onClick : undefined}>
+            <Header
+              title={title}
+              titleAs={titleAs}
+              description={description}
+              header={header}
+              expanded={opened}
+              isSection
+            />
+          </ContentWrapper>
           {actions && <Actions actions={actions} />}
```

Comment:
In this case, when `actions` are set both with `onClick`, it's not possible to click on the clickable element in the `actions`  😬